src/adapters/local.ts

```diff
@@ -42,11 +42,24 @@ export default class LocalAdapter implements IAdapter {
     }
 
     async rename(oldPath: string, newPath: string): Promise<boolean> {
-        throw new Error('rename NYI');
+        await fs.renameAsync(this.fullPath(oldPath), this.fullPath(newPath));
+
+        return true;
```

Comment:
yeah I know, its just a generic.

ISomething comes from .net - I personally prefer it.

so are we doing a boolean or metadata?